src/Feedback.Analyzer.Backend/Feedback.Analyzer.Infrastructure/Common/Prompts/Brokers/PromptExecutionBroker.cs

```diff
@@ -0,0 +1,15 @@
+using Feedback.Analyzer.Application.Common.Prompts.Brokers;
+using Feedback.Analyzer.Domain.Common.Exceptions;
+using Feedback.Analyzer.Domain.Extensions;
+using Microsoft.SemanticKernel;
+
+namespace Feedback.Analyzer.Infrastructure.Common.Prompts.Brokers;
+
+public class PromptExecutionBroker(Kernel kernel) : IPromptExecutionBroker
+{
```

Comment:
Add type docs  :

"Provides prompt execution broker functionality and mapping arguments and results"